packages/docs/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -132,6 +132,7 @@ export default {
       transitionTimingFunction: 'ease-out',
       transitionDuration: '.2s',
       borderRadius: 2,
+      outline: 'none',
```

Comment:
Haven't looked at this in context yet, but how would this work for keyboard users? This looks like it breaks accessibility at first glance

Reply:
Yeah you're right, I've realised that aside from this bug, accessibility seems to be broken in Safari. I'll open a new issue

Reply:
Right so it *is* working, though it needs to manually be enabled in preferences. That just leaves the original glitch to be fixed by some other means :)